include/mbedtls/mbedtls_config.h

```diff
@@ -530,7 +530,7 @@
  * Uncommenting this macro removes support for AES operations that use 192-
  * or 256-bit keys.
  *
- * Tradeoff: Uncommenting this macro reduces ROM footprint by ~1116 bytes.
+ * Tradeoff: Uncommenting this macro reduces the size of AES code by about 4%.
```

Comment:
I think this is a good change. Is it worth going a bit further and stating the architecture for which this 4% is relevant? I know lots of factors go into code size and you're deliberately not being super specific here (which I agree with) but mentioning the architecture might be a nice detail to have. Your call, I won't "request changes" based on this point alone.

Reply:
Thanks Tom. I have stated the architecture which measures the code size.

I also removed `Tradeoff` in the original comment. AFAIU, uncommenting this macro only removes feature of `192- and 256-bit keys` in AES. There is no performance hit. So I think `Tradeoff` is unclear then I removed it. Additionally, I have paraphrased comment, let me know if it's not appropriate.